frontend/src/components/authForms/LoginForm.js

```diff
@@ -1,46 +1,57 @@
 import { useState, useContext } from "react";
 import { MyContext } from "../../context/context";
 import { useNavigate } from "react-router";
-import { useHttp } from "../../customHooks/useHttp";
+import axios from "axios";
 export const LoginForm = () => {
   const [username, setUsername] = useState("");
   const [password, setPassword] = useState("");
-  const { data, loading, error, fetchData } = useHttp(
-    "http://localhost:8080/api/v1/auth/login",
-    "POST"
-  );
+  const [responseMsg, setResponseMsg] = useState(null);
   const cx = useContext(MyContext);
   const navigate = useNavigate();
+
   const handleLogin = async (event) => {
     event.preventDefault();
-    fetchData({ username, password });
-
-    // const data =  res.json();
-
-    // console.log(data);
-    // cx.setLoggedIn(true);
-    // localStorage.setItem("token", data.token);
-    // navigate("/userfeed");
+    const access_token = localStorage.getItem("access_token");
+    try {
+      const response = await axios.post(
+        `http://localhost:8080/api/v1/auth/login`,

```

Comment:
2 things
1. You can create axios service where you will define get post put and delete structure and define default URL (for example http://localhost:8080/api/v1 is part of every endpoint so no need to write it every time
2. You can create services for those request where you will define how each request should be executed